DOC - clean up howto for release (#2047)
skrub-data/skrub: doc/guides/table_report/exporting.rst

    tr = TableReport(df)
    json_data = tr.json()

Note that this will export all parts of the |TableReport|, including the distribution

> plots in SVG format if they have been generated. To avoid this, plots should be

Reviewer: ```suggestion
plots in SVG format if they have been generated. If you do not need them, plots should be
```
